zdm/misc_functions.py

```diff
@@ -1901,10 +1904,12 @@ def plot_zdm_basic_paper(zDMgrid,zvals,dmvals,zmax=1,DMmax=1000,norm=0,log=True,
 	plt.tight_layout()
 	
 	plt.savefig(name)
+	plt.title(title+str(H0))
+    plt.show()
 	plt.close()		
 
 
-def plot_grid_2(zDMgrid,zvals,dmvals,zmax=1,DMmax=1000,norm=0,log=True,name='temp.pdf',label='$\\log_{10}p(DM_{\\rm EG},z)$',project=False,conts=False,FRBZ=None,FRBDM=None,Aconts=False,Macquart=None):
+def plot_grid_2(zDMgrid,zvals,dmvals,zmax=1,DMmax=1000,norm=0,log=True,name='temp.pdf',label='$\\log_{10}p(DM_{\\rm EG},z)$',project=False,conts=False,FRBZ=None,FRBDM=None,Aconts=False,Macquart=None,title="Plot",H0=cos.DEF_H0):
```

Comment:
ditto re the title! Also a quick heads-up: in papers, you almost never add a title to a plot, you just describe in with a caption. Absolutely fine to have a title option, but something to keep in mind for the future paper.

Reply:
Okay, will keep that in mind! I will change these then...it's just that the titles help me when I'm actually obtaining the plots